wallet/createtx.go

```diff
@@ -478,7 +478,7 @@ func (w *Wallet) txToMultisigInternal(op errors.Op, dbtx walletdb.ReadWriteTx, a
 	}
 
 	msgtx := wire.NewMsgTx()
-	scriptSizers := []txsizes.ScriptSizer{}
+	scriptSizers := []int{}
```

Comment:
should replace "sizer" with "size", "sizers" with "sizes", etc. throughout this PR.